pyracing/response_objects/chart_data.py

```diff
@@ -0,0 +1,64 @@
+from ..constants import ChartType, Category, License
+from datetime import datetime
+
+
+class ChartData:
+    def __init__(self, category, type, list):
+        self.category = category  # oval, road, dirt_road, dirt_oval
+        self.type = type  # irating, ttrating, or license_class
+        self.list = list  # A list of Iratings, TTRatings, or LicenseClasses
+
+    def current(self):
+        return self.list[-1]
+
+    def type_string(self):
+        return ChartType().number_to_string(self.type)
+
+    def category_string(self):
+        return Category().number_to_string(self.category)
+
+
+class IRating:
+    def __init__(self, tuple):
+        self.datetime = self.datetime_from_iracing_timestamp(tuple[0])
+        self.value = tuple[1]
+
+    # iRacing has all of their timestamps in ms so we need to divide
+    @staticmethod
+    def datetime_from_iracing_timestamp(timestamp):
```

Comment:
Now that these are in the same file, we could pull this method out to be static and use the same method on all 3 of `IRating`, `TTRating`, and `LicenseClass`